docs/snippets/angular/build-storybook-production-mode.with-builder.js.mdx

```diff
@@ -0,0 +1,6 @@
+```shell
+# Builds Storybook with Angular's custom builder
+# See https://storybook.js.org/docs/angular/get-started/install 
+# to learn how to create the custom builder
+ng run my-project:build-storybook
```

Comment:
we might be able to just instruct users to add this to their `"scripts"` tag and then the rest of the instructions could stay the same. WDYT?

```json
{
  "scripts": {
     "build-storybook": "ng run my-project:build-storybook",
  }
}
```

Reply:
I've updated the Publish Storybook and factored it in with the latest commit.